src/index.ts

```diff
@@ -1,5 +1,7 @@
-import * as colors from "./colors";
-import * as fonts from "./fonts";
+import * as colors from './colors';
+import * as fonts from './fonts';
+import {Table} from './Table';
 
-export { colors };
-export { fonts };
+export {colors};
```

Comment:
I was being lazy not adding any CI to this project. If we're going to start having whitespace and quote style changes, it's time for CI and a prettier config.